ros2/ament.bzl

```diff
@@ -147,6 +147,18 @@ def _ros2_ament_setup_rule_impl(ctx):
         )
         outputs.append(dynamic_library)
 
+    for dep in ctx.attr.idl_deps:
```

Comment:
Calling .to_list is inefficient. BTW, I think you'll need:

```
    idls = depset(
        transitive = [
            depset([
                struct(
                    package_name = dep.label.name,
                    srcs = dep[Ros2InterfaceInfo].info.srcs
                )
            ])
            for dep in ctx.attr.idl_deps
        ],
    ).to_list()
```
and then you that struct in the loop.

Otherwise, nice work! :+1: